src/platform_impl/macos/monitor.rs

```diff
@@ -158,6 +158,27 @@ pub fn primary_monitor() -> MonitorHandle {
   MonitorHandle(CGDisplay::main().id)
 }
 
+// `from_point` get a monitor handle which contains the given point.
+pub fn from_point(x: f64, y: f64) -> Option<MonitorHandle> {
+  unsafe {
+    for monitor in available_monitors() {
+      let bound = CGDisplayBounds(monitor.0);
+      // dumb way. a better way is to use [CGRectContainsPoint](https://developer.apple.com/documentation/coregraphics/1456316-cgrectcontainspoint)
+      // which is not supported by core-graphics binding yet.
+      // Once the [PR](https://github.com/servo/core-foundation-rs/pull/526) is accepted
```

Comment:
you can inline the implementation of that PR here instead. cocoa-rs will not release anytime soon.

Reply:
done